src/load_balancer.rs

```diff
@@ -42,8 +45,17 @@ impl LoadBalancer {
         self.algorithm = algorithm
     }
 
+    pub async fn show_table(&self) -> Result<(), LoadBalancerError> {
+        let df = self.ctx.sql(&format!("select * from {DF_TABLE_NAME} order by id")).await?;
```

Comment:
I would recommend storing all queries in separate files later, it's easier to maintain that way.

Reply:
Good advice as amount of sql queries is getting bigger